tests/observeChildren.html

```diff
@@ -29,8 +29,8 @@
   suite('observeChildren', function() {
 
   function makeShadowRootWithSlot(element) {
-    element.attachShadow({mode: 'open'});
-    element.shadowRoot.appendChild(document.createElement('slot'));
+    ShadyDOM.wrap(element).attachShadow({mode: 'open'});
```

Comment:
Instead of refactoring all tests to always use wrapping, I would rather do the following:

```js
function wrapIfPatched(element) {
  if (window.location.search.match('noPatch')) {
    return ShadyDOM.wrap(element);
  }

  return element;
}
```

That way we know the old version remains working as well.

Reply:
`ShadyDOM.wrap` does this internally.